RetailCoder.VBE/Common/ICodeModuleExtensions.cs

```diff
@@ -0,0 +1,35 @@
+﻿using Antlr4.Runtime;
+using Rubberduck.Parsing.Symbols;
+using Rubberduck.VBEditor.SafeComWrappers.Abstract;
+
+namespace Rubberduck.Common
+{
+    public static class ICodeModuleExtensions
```

Comment:
The `I` prefix indicates an interface; I'd rename this one to `CodeModuleExtensions`; this is similar to LINQ's `Enumerable` static class, which extends the `IEnumerable` interface.